spot_utils/structures/image.py

```diff
@@ -0,0 +1,57 @@
+"""Data structures for images and pointclouds."""
```

Comment:
general question for here and elsewhere: to what extent should we be introducing our own data structures vs. relying on existing ones from our dependencies?

in this file, I think open3d has a number of classes that we could potentially use (e.g. http://www.open3d.org/docs/release/python_api/open3d.geometry.PointCloud.html)

Reply:
I didn't want to commit to open3d quite yet since there are a few other options and it would be good not to mix. We should discuss.